tests/pytests/integration/ssh/test_log.py

```diff
@@ -0,0 +1,122 @@
+"""
+Integration tests for salt-ssh py_versions
+"""
+import logging
+
+import pytest
+from saltfactories.utils import random_string
+
+from tests.support.helpers import Keys
+
+pytest.importorskip("docker")
+
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.slow_test,
+    pytest.mark.skip_if_binaries_missing("dockerd"),
+]
+
+
+@pytest.fixture(scope="module")
+def ssh_auth():
+    return random_string("sshpassword"), "app-admin"
+
+
+@pytest.fixture(scope="module")
+def ssh_keys(tmp_path_factory):
+    """
+    Temporary ssh key fixture
+    """
+    with Keys(tmp_path_factory) as keys:
+        yield keys
+
+
+@pytest.fixture(scope="module")
+def ssh_docker_container(salt_factories, ssh_keys, ssh_auth):
+    """
+    Temporary docker container with python 3.6 and ssh enabled
+    """
+    ssh_pass, ssh_user = ssh_auth
+    container = salt_factories.get_container(
+        random_string("ssh-py_versions-"),
+        "ch3ll/salt-setup:ssh-password1",
```

Comment:
**question** it's probably fine, at least for now, but is it possible to use the saltstack docker org or something?

Reply:
Yeah, thanks for pointing that out. I meant to update it when it was available in our docker github. And now it is. Thanks!